### ISSUE TITLE
Proc macro from `refinery` crate not expanded

### ISSUE THREAD

**Issue by winterqt** [rust-analyzer/rust-analyzer#10116]
**RA version**: `2021-08-16`
**rustc version**: `1.53.0`

# Reproduction steps
1. Make a new project
2. Add `refinery` with the `rusqlite` feature enabled and `rusqlite` to your dependencies
3. `main.rs`:
```rust
use rusqlite::Connection;

mod embedded {
    use refinery::embed_migrations;
    embed_migrations!("./tests/sql_migrations");
}

fn main() {
    let mut conn = Connection::open_in_memory().unwrap();
    embedded::migrations::runner().run(&mut conn).unwrap();
}
```
4. Observe `proc macro not expanded` diagnostic under the `embed_migratons` invocation

I can confirm that proc macros are enabled.

**Comment by winterqt** [comment]
Seems to affect all proc macros, and this is the only error I can find in my logs:

```
Failed to find proc macros. Error: ExpansionError(
    "Cannot create expander for [REDACTED]: Io(Custom { kind: InvalidData, error: \"no commit info\" })",
)
```

**Comment by lnicola** [comment]
Should have been fixed in #9936. Try upgrading.

**Comment by winterqt** [comment]
Thanks, that worked.
